src/common/modules/popup/notice/NoticePopup.vue

```diff
@@ -35,21 +36,26 @@ export default {
             popupList: [] as Array<NoticePostModel>,
         });
 
+
+        const apiQuery = new ApiQueryHelper();
+        const getQuery = () => {
+            apiQuery
+                .setFilters([{
+                    k: 'name',
+                    v: 'console:board',
+                    o: '',
+                }, {
+                    k: 'data.show_popup',
+                    v: false,
+                    o: '=',
+                }]);
+            return apiQuery.data;
+        };
```

Comment:
이건 유저의 입력에 의해서 변경하는 것이 아닌것 같네요!
호출할 때마다 filter setting 해줄 필요 없이, 고정으로 박아버려도 될것같습니다~

```suggestion
        const apiQuery = new ApiQueryHelper().setFilters([{
                    k: 'name',
                    v: 'console:board',
                    o: '',
                }, {
                    k: 'data.show_popup',
                    v: false,
                    o: '=',
                }]);
```